official/vision/beta/projects/yolo/utils/math_ops.py

```diff
@@ -0,0 +1,66 @@
+"""A set of private math operations used to safely implement the YOLO loss."""
```

Comment:
Can you revert? I thought you'll rename math_ops to utils. But I feel math_ops name is fine.

Reply:
yes we can revert this.